lib/ivar.ex

```diff
@@ -121,6 +121,22 @@ defmodule Ivar do
   def delete(url, opts \\ []), do: Ivar.new(:delete, url, opts)
 
   @doc """
+  Calls `Ivar.new/3` with the method set to `:options` for the given `url` and `opts`
+  
+  Args
+  
+    * `url` - a binary containing the full url (e.g. `https://example.com`)
+    * `opts` - keyword list containing any valid options for the configured http adapter
+    
+  Usages
+  
+      Ivar.delete("https://example.com", [timeout: 10_000])
+      %{method: :delete, url: "https://example.com", opts: [timeout: 10_000]}
+  """
+  @spec delete(binary, Keyword.t) :: map
```

Comment:
Just need to update the docs/spec attribute here to say options instead of delete